honeypotio/staticpages
# Growth: Homepage speed optimizations [#2089]

- Removed heap
- Removed optimizely for now

**How to Test**
- Visit [google page insights](https://developers.google.com/speed/pagespeed/insights/)
- Paste http://staging.honeypot.co/pr-444/ which where this staging server is located
- It should be green for both mobile & desktop

ticket: https://github.com/honeypotio/honeypot/issues/2089
